docs/Compatibility.md

```diff
@@ -4,156 +4,247 @@ The Ruby Datadog Trace library is open source. See the [dd-trace-rb][1] GitHub r
+
 [26]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#mongodb
+
 [27]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#mysql2
+
 [28]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#nethttp
+
 [29]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#opensearch
+
 [30]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#postgres
+
 [31]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#presto
+
 [33]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#que
+
 [34]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#racecar
+
 [35]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rack
+
 [36]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rails
+
 [37]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rake
+
 [38]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#redis
+
 [39]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#resque
+
 [40]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rest-client
+
 [41]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#roda
+
 [42]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sequel
+
 [43]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#shoryuken
+
 [44]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sidekiq
+
 [45]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sinatra
+
 [46]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sneakers
+
 [47]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#stripe
+
 [48]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sucker-punch
 
+### Support Policy
+
+Datadog APM for Ruby is built upon dependencies defined in specific versions of the host operating system, Ruby
+runtime, certain Ruby libraries, and the Datadog Agent/API. When these versions are no longer supported by their
+maintainers, Datadog APM for Ruby limits its support for these as well.
+
+#### Levels of Support
+
+| **Level**                                              | **Support provided**                                                                                                                                                         |
+|--------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Unsupported</span>      | No implementation. [Contact our customer support team for special requests.](https://www.datadoghq.com/support/)                                                             |
```

Comment:
> Unsupported: No implementation

I dont' quite grok this.

In fact throughout the table I feel like making reference to no/initial/full implementation/features very much muddies the point.

To me, both of the following feel much more synthetic and clearer at the same time:

https://guides.rubyonrails.org/maintenance_policy.html
https://nodejs.org/en/about/previous-releases

Reply:
@ekump your interpretation is correction. This was meant to address things that might be asked, but is not yet (or might never be) supported.

I'm fine with the name `Unimplemented` instead, if that seems clearer.

Reply:
updated so there is both `unsupported` and `unimplemented`.